lib/timers.js

```diff
@@ -439,7 +439,7 @@ exports.clearInterval = function(timer) {
 };
 
 
-const Timeout = function(after) {
+function Timeout(after) {
```

Comment:
Why even assign it though? It's unnecessary.

Typically we have avoided modernization unless it meant performance increases or was part of some refactor that touched the code a lot anyways.